### DISCUSSION TITLE
Upload my mod into github?

### DISCUSSION THREAD

**Discussion by Danimal696** [flareteam/flare-game#884] [General]
A few players told me to upload to github. How should i go about it? I guess I want it to always be compatible with newest engine but some iterations totally broke it. So maybe it´s better to anchor it to a specific version? Do i make a fork of the main project or start a new one?. Will it remain visible with the rest of the Flare database if so?. What advantages does it give me uploading to github (download count?)?

**Comment by WithinAmnesia** [comment]
I wish I better knew how to do this too. I am still learning how github operates myself. Do all ~'big' modders get a fork in that case if we tie community mods to the Flare github?

**Comment by dorkster** [comment]
We try to keep Flare forward compatible when possible. So if something breaks with a new version, that is most likely on us and you should submit a bug report.

It would only make sense to use Github if you were already using Git for version control while developing your mod. You would only fork if you wanted to contribute *directly* to flare-game. Where you are creating your own mod, you would be creating your own repository that is separate from all of flareteam.

Of course, if you don't care about Git or Github, you can always just host a ZIP of your mod somewhere.

Whatever you choose, just make me aware of it so I can add it to the Mods section on the Flare website.
